### PR TITLE
Support `distributed` keyword for Swift

### PR DESCRIPTION
**Pull request by mportiz08**
This new syntax is in support of the ["Distributed Actor Isolation"][1] feature that has been accepted for Swift 5.7.

[1]: https://github.com/apple/swift-evolution/blob/main/proposals/0336-distributed-actor-isolation.md

### Changes

New Swift language keyword: `distributed`

### Checklist
- [X] Added markup tests, or they don't apply here because...
- [x] Updated the changelog at `CHANGES.md`

### CODE REVIEW

**Comment by mportiz08** [comment]
@joshgoebel I noticed that the linter check is failing, but the issues seem to stem from files not touched by this PR, and I believe the same check is failing on `main`.

Let me know if I missed anything there though, thanks!

File: CHANGES.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 Grammars:
 
 - enh(swift) add SE-0335 existential `any` keyword (#3515) [Bradley Mackey][]
+- enh(swift) add support for `distributed` keyword [Marcus Ortiz][]
 
 [Bradley Mackey]: https://github.com/bradleymackey
```
Comment (joshgoebel) [inline CHANGES.md:8]: You need to add a link ref for your name here as well, it doesn't matter if it's way down below somewhere, each release needs their own link references.

**Review by joshgoebel — CHANGES_REQUESTED** [review]
Otherwise looks good.

File: CHANGES.md (hunk 1 above)
Comment (mportiz08) [reply CHANGES.md:8]: Got it, thanks. [Updated](https://github.com/highlightjs/highlight.js/pull/3523/commits/69eb3f1ddbcdf85f55477db1542ee7a9f4340b5f)

**Review by joshgoebel — APPROVED** [review]

**Pull request merged**